### ISSUE TITLE
Add basic pre-flight checks and setup (Linux / Virtualbox)

### ISSUE THREAD

**Issue by gbraad** [code-ready/crc#47]
Depends on #9 

Implement pre-flight checks and setup related to use of Virtualbox on Linux

  * [ ] Check if Virtualbox installed
  * [ ] Check if configured
  * ...

**Comment by gbraad** [comment]
WONTFIX
